### ISSUE TITLE
[Chromaticraft] Teleportation pump in a different dimension causes periodic lag spikes if not explicitly chunkloaded

### ISSUE THREAD

**Issue by DoomFruit** [ReikaKalseki/Reika_Mods_Issues#3070]
I usually set up a pump in the Nether to feed lava into my stores. Previously, I'd have used the ender-thermic pump from Extra Utilities, but now that I'm giving Chromaticraft a try, I noticed that it has its own one. I set that up, set it running and it's producing a lot of fluid for me. So far, so good. The problem is that it causes periodic spikes of CPU usage, bringing my tick rate down to 5 or 6 (and tick time up to 150 ms as shown by /forge tps) from a base rate of 15-20 ms. Attached are a pair of images taken by running Opis. The first shows the tick time as a function of time and the second shows what's actually eating CPU time. Right at the top, with 16 ms in one single item is my singular teleportation pump.

Mod versions: Rotarycraft, Reactorcraft, Electricraft, Chromaticraft, DragonAPI are all v31b. There are a huge number of other mods installed (it's a large kitchen sink 1.7.10 pack, I can supply the list on request).

![CC-TeleportPumpLagSpike](https://user-images.githubusercontent.com/9013415/187878550-02da5bcd-9caf-4437-ae83-882b6e3ef62e.png)
![CC-TeleportPumpLagSpikeNumbers](https://user-images.githubusercontent.com/9013415/187878588-3afa3da5-012a-482f-9af6-bfbdceb379ab.png)

**Comment by DoomFruit** [comment]
An update: after going into the Nether to check on some things, I noticed that the lag spikes dropped to a much lower level when I was in the same dimension as the pump, but out of render/chunkloading range (400 metres away). Something like 80-100 ms per tick. When I got within render range of the pump (read: standing on top of it), the lag spikes disappeared entirely, instead smoothing out to a consistent 20 ms/tick.

Then I tried putting a chunkloader block next to the pump. I know that it is a chunkloader on its own (it wouldn't work unattended otherwise), but it was worth a try. I used a ChickenChunks loader with zero radius - so it was loading just the pump and nothing else. Going back to the overworld had the same result as standing on the teleportation pump - no lag spikes at all.

So a more accurate description would be: teleportation pump in a different dimension which is not explicitly chunkloaded causes periodic lag spikes.
